core/shared/src/main/scala/org/http4s/headers/Cookie.scala

```diff
@@ -28,23 +28,24 @@ object Cookie {
   def apply(head: RequestCookie, tail: RequestCookie*): `Cookie` =
     apply(NonEmptyList(head, tail.toList))
 
-  def parse(s: String): ParseResult[Cookie] =
-    ParseResult.fromParser(parser, "Invalid Cookie header")(s)
+  def parse(s: String): ParseResult[Cookie] = {
+    val (errors, cookies) = s
+      .split("; |;")
```

Comment:
I erred earlier saying this was like comma separated headers and more general.  This also means it has a simpler grammar than the standard header list separation.  I think this is right and I was wrong.  Cookies are weird.